src/python/pants/backend/codegen/thrift/scrooge/additional_fields.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+from pants.backend.codegen.thrift.target_types import (
+    ThriftSourcesGeneratorTarget,
+    ThriftSourceTarget,
+)
+from pants.engine.target import BoolField
+
+
+class ScroogeFinagleBoolField(BoolField):
+    alias = "finagle"
+    default = False
+    help = "If True, then generate Finagle classes when using Scrooge as the Thrift generator."
```

Comment:
Is this "also" generate Finagle? Or "instead" generate Finagle?

Reply:
"also" - This field is similar in spirit to the `grpc` field for protobuf.

Reply:
Okay, how about saying "If True, then also generate Finagle classes..."?